times.py

```diff
@@ -123,6 +123,9 @@ def time_app_link(self: 'Runtime') -> List[str]:
       else:
         durations = Runtime.get_durations_fenix(f, print_lines)
 
+    if len(durations) == 0:
+      raise ValueError('Did not find any matching lines for variant: is the intent configured correctly?')
```

Comment:
I think this would make more sense in `calculate_average_app_link`, since then if we change things around to call `calculate_average_app_link` elsewhere, we still catch this error.

Reply:
While that's safer, conceptually `time_app_link` is the function that knows it's reading lines and there's a possibility that those lines won't exist so it should check for that (i.e. I think the current implementation makes sense too).

I don't feel too strongly, given the time constraints, so tell me what to do. :P

Reply:
@mcomella I'd say move it to `calculate_average_app_link` since that's where the logic error actually occurs and it's more obvious (to me) with it next to the calculation. Though I suppose it doesn't matter too much since this could probably use some refactoring later on anyway...